ksql-clickstream-demo/Dockerfile

```diff
@@ -14,9 +14,9 @@ ENV KSQL_CLASSPATH=/usr/share/java/${ARTIFACT_ID}/${ARTIFACT_ID}-${PROJECT_VERSI
 ENV KSQL_CONFIG_DIR="/etc/ksql"
 ENV KSQL_LOG4J_OPTS="-Dlog4j.configuration=file:/etc/ksql/log4j-rolling.properties"
 
-RUN wget -q https://s3-us-west-2.amazonaws.com/jenkins-confluent-packages/4.1.x/88/archive/4.1/confluent-oss-4.1.0-SNAPSHOT-2.11.tar.gz \
-    && tar xzvf confluent-oss-4.1.0-SNAPSHOT-2.11.tar.gz --strip-components 1 \
-    && rm confluent-oss-4.1.0-SNAPSHOT-2.11.tar.gz
+RUN wget -q http://packages.confluent.io/archive/5.0/confluent-oss-5.0.0-beta1-2.11.tar.gz \
```

Comment:
Why the change to http?

Reply:
I don't believe that was intentional. Will change it back.

Reply:
cc @rodesai

Reply:
done.